src/components/text/text-sign.vue

```diff
@@ -37,32 +37,27 @@ export default class SignComponent extends Vue {
     }
 
     private clickable(si: SignInterpretation) {
-        if (!si.character) {
-            return false;
-        }
-        if (this.cssStrings(si).indexOf('is-reconstructed-true') !== -1) {
-            return false;
-        }
-        return true;
+        return si.character && !si.isReconstructed;
     }
 }
 </script>
 
 <style lang="scss" scoped>
+@import "../../assets./styles/_variables.scss";
```

Comment:
My computer yelled at me that this is invalid (this is on Ubuntu 18.04).  I changed to `@import "@/assets/styles/_variables.scss";` and that seemed to make everyone happy.